src/Node/DocBlock/CustomTags/ModuleTag.php

```diff
@@ -19,7 +19,7 @@ class ModuleTag extends BaseTag implements StaticMethod
     /**
      * @var string
      */
-    protected $name = 'module';
+    protected string $name = 'module';
```

Comment:
[Note] Fine for merge-only,g it is an internal BC-break,  because the module is not stable